app/views/RoomsListView/index.js

```diff
@@ -634,21 +625,28 @@ export default class RoomsListView extends React.Component {
 
 		return (
 			<SafeAreaView style={styles.container} testID='rooms-list-view' forceInset={{ bottom: 'never' }}>
-				<StatusBar />
-				{this.renderScroll()}
-				{showSortDropdown
-					? (
-						<SortDropdown
-							close={this.toggleSort}
-							sortBy={sortBy}
-							groupByType={groupByType}
-							showFavorites={showFavorites}
-							showUnread={showUnread}
-						/>
-					)
-					: null
-				}
-				{showServerDropdown ? <ServerDropdown /> : null}
+				<PanGestureHandler
+					onHandlerStateChange={this.onHandlerStateChange}
+					minDeltaY={20}
+				>
+					<Animated.ScrollView style={styles.scrollView}>
```

Comment:
No need for this Animation logic here.
This offset already works as expected: https://github.com/RocketChat/Rocket.Chat.ReactNative/blob/develop/app/views/RoomsListView/index.js#L602
If you remove iOS condition, it's probably going to work on Android.

Reply:
The Animation is needed only to detect the swipe, but I will send one improvement.

Reply:
This search already works on iOS using offset.
Your PR doesn't.
Offset is not an optimal solution, but it's fine.
Wrapping everything in another ScrollView is error prone.
Can we talk on https://open.rocket.chat?
I'm `@diego.mello`.
Thanks :)